script/licensed

```diff
@@ -40,7 +40,7 @@ OptionParser.new do |opts|
   end
 end.parse!
 
-source = Licensed::Source::Filesystem.new(module_path || "vendor/grammars/*/", type: "grammar")
+source = Licensed::Source::Filesystem.new(module_path || "#{File.expand_path("../", File.dirname(__FILE__))}/vendor/grammars/*/", type: "grammar")
```

Comment:
FYI in local testing, I had to remove the trailing "/" from the file path to properly find license data 🤷‍♂️

Reply:
:+1: Once we sort out the Licensee part of things, I'd like to steal a bit more of your time and take you up on your suggestion of switching this to use `Licensed::Source::Manifest`, but only when you've got the time - I still need to give it a go too 😄